lib/router.js

```diff
@@ -11,25 +11,21 @@ define(['helper'], function (helper) {
     var running = false;
 
     function saveState() {
-      var e = [];
+      var e = '#!';
 
-      if (currentView) {
-        e.push('v:' + currentView);
-      }
+      e += 'v:' + currentView;
 
       if (currentObject) {
         if ('node' in currentObject) {
-          e.push('n:' + encodeURIComponent(currentObject.node.nodeinfo.node_id));
+          e += ';n:' + encodeURIComponent(currentObject.node.nodeinfo.node_id);
         }
 
```

Comment:
to be sure, a `else if` instatt of a only `if` - both states are not possible at the same time.